styles/style.css

```diff
@@ -326,3 +422,119 @@ footer {
 .info > p {
   line-height: 140%;
 }
+
+.big {
+  display: none;
+}
+
+.bottombar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+@media only screen and (min-width: 700px) {
```

Comment:
To target desktop screens, we need a value between `992px and 1100px` , anything below that range is too small

- [ ] Kindly update media query 🙂